packages/dapp-svelte-wallet/api/src/wallet.js

```diff
@@ -49,167 +50,150 @@ export function buildRootObject(_vatPowers) {
           },
         });
-        return {
-          type: 'walletOfferAdded',
-          data: await wallet.addOffer(
-            { ...data, actions },
-            { ...meta, dappOrigin },
-          ),
-        };
-      }
-      case 'walletDeclineOffer': {
-        return {
-          type: 'walletDeclineOfferResponse',
-          data: await wallet.declineOffer(data),
-        };
-      }
-      case 'walletCancelOffer': {
-        return {
-          type: 'walletCancelOfferResponse',
-          data: await wallet.cancelOffer(data),
-        };
-      }
-      case 'walletAcceptOffer': {
-        const { outcome } = await wallet.acceptOffer(data);
-        // We return the outcome only if it is a string. Otherwise we
-        // return a default message.
-        const result =
-          typeof outcome === 'string'
-            ? { outcome }
-            : { outcome: 'Offer was made.' };
-        return {
-          type: 'walletAcceptOfferResponse',
-          data: result,
-        };
-      }
+        return notifier;
+      },
+      async getDepositFacetId(brandBoardId) {
+        await approve();
+        return walletInternals.getDepositFacetId(brandBoardId);
+      },
+      async suggestIssuer(petname, boardId) {
+        await approve();
+        return walletInternals.suggestIssuer(petname, boardId, dappOrigin);
+      },
+      async suggestInstallation(petname, boardId) {
+        await approve();
+        return walletInternals.suggestInstallation(
+          petname,
+          boardId,
+          dappOrigin,
+        );
+      },
+      async suggestInstance(petname, boardId) {
+        await approve();
+        return walletInternals.suggestInstance(petname, boardId, dappOrigin);
+      },
+    };
+    return harden(bridge);
+  };
 
-      case 'walletGetOffers':
-      case 'walletGetOfferDescriptions': {
-        const result = await wallet.getOffers({ origin: dappOrigin });
-        return {
-          type: 'walletOfferDescriptions',
-          data: result,
-        };
-      }
+  /** @type {WalletBridge} */
+  const anonymousBridge = {
+    addOffer(offer) {
+      return walletInternals.addOffer(offer);
+    },
+    getDepositFacetId(brandBoardId) {
+      return walletInternals.getDepositFacetId(brandBoardId);
+    },
+    async getOfferNotifier() {
+      return inboxNotifier;
+    },
+    getPursesNotifier() {
+      return walletInternals.getPursesNotifier();
+    },
+    suggestInstallation(petname, installationBoardId) {
+      return walletInternals.suggestInstallation(petname, installationBoardId);
+    },
+    suggestInstance(petname, instanceBoardId) {
+      return walletInternals.suggestInstance(petname, instanceBoardId);
+    },
+    suggestIssuer(petname, issuerBoardId) {
+      return walletInternals.suggestIssuer(petname, issuerBoardId);
+    },
+  };
+  harden(anonymousBridge);
 
-      default: {
-        return false;
-      }
-    }
+  async function getWallet() {
```

Comment:
If this facet is meant to be used in the REPL and the deploy scripts, it's leaving out a lot. I would expect users to be saving issuers, depositing payments, and such.

Reply:
As discussed, this doesn't leave anything out (all is available from current `home.wallet~.getInternals`), it just makes space for a more careful API that we still have to define.

Reply:
It's now `home.wallet~.getAdminFacet()` to explain that these are operations that typical users will not want (until we expose those operations in a more carefully-thought-out API).